services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -549,4 +682,12 @@ meters in flow rules at all (#data.flowType flow)"() {
             path[1..-2].every { it.switchId.description.contains("OF_12") }
         }
     }
+
+    void verifyBurstSizeOnWb5164(Long expected, Long actual) {
+        assert Math.abs(expected - actual) <= expected * 0.01
```

Comment:
check whether the `0.01` coef is configurable in kilda. If yes - use the same config value as kilda does (in template)

Reply:
it is not configurable
it is just variable in `switchmanager`